core/execution/future/market.go

```diff
@@ -1355,6 +1360,24 @@ func (m *Market) uncrossOnLeaveAuction(ctx context.Context) ([]*types.OrderConfi
 
 	// send order events in a single batch, it's more efficient
 	m.broker.SendBatch(evts)
+
+	// after auction uncrossing we can relax the price requirement and release some excess order margin if any was placed during an auction.
+	for k, d := range m.partyMarginFactor {
+		if !d.IsZero() {
+			partyPos, _ := m.position.GetPositionByPartyID(k)
+			if partyPos != nil && (partyPos.Buy() != 0 || partyPos.Sell() != 0) {
+				marketObservable, mpos, increment, _, _, orders, err := m.getIsolatedMarginContext(partyPos, nil)
+				if err != nil {
+					continue
+				}
```

Comment:
Could anything happen here if we are just ignoring this error?

Reply:
I don't think it's really possible for an error to occur here so the continue is definitely not a problem. panic here would be a bit too much in my opinion.